test/integration/image-optimization/next.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  target: 'serverless',
+  experimental: { optimizeImages: true },
```

Comment:
This needs the serverless server https://github.com/vercel/next.js/blob/canary/test/integration/font-optimization/server.js